### PR TITLE
feat: support proxied cleanup method

### PR DESCRIPTION
**Pull request by git-qfzhang**

### CODE REVIEW

File: docs/Usage/proxied.md [hunk 1]
```diff
@@ -147,7 +147,7 @@ Proxied 命令为我们提供了三个子命令：
     
     Example
     
-      Just clean.   $ s proxied clean
+      Just cleanp.   $ s proxied cleanup
```
Comment (rsonghuster) [inline docs/Usage/proxied.md:150]: typo

File: src/command/proxied.ts [hunk 2]
```diff
@@ -13,6 +13,9 @@ export async function invoke(inputs: IInputs): Promise<any> {
 export async function clean(inputs: IInputs): Promise<any> {
   return await componentMethodCaller(inputs, 'devsapp/fc-proxied-invoke', 'clean');
 }
+export async function cleanup(inputs: IInputs): Promise<any> {
```
Comment (rsonghuster) [inline src/command/proxied.ts:16]: 还保留 clean 是为了保持兼容吧

Comment (git-qfzhang) [reply src/command/proxied.ts:16]: 对

File: docs/Usage/proxied.md (hunk 1 above)
Comment (git-qfzhang) [reply docs/Usage/proxied.md]: done

**Pull request merged**